frontend/src/components/HelloWorld.vue

```diff
@@ -0,0 +1,119 @@
+<template>
```

Comment:
Can we rename this class to be more specific to our project?

Reply:
File not required. Hence removed.